HARMONIZED_TEST_ENVIRONMENT.md

```diff
@@ -0,0 +1,72 @@
+Harmonized Xroad test environment
+===================
+
+- Common test development environment to Finland and Estonia
+	- Includes servers: CA, CENTRAL SERVER, MANAGEMENT SECURITY SERVER, SECURITY SERVERS
```

Comment:
Should we add link to installation instructions e.g. https://github.com/ria-ee/X-Road/tree/develop/ansible?

Reply:
added